checker/jtreg/stubs/issue1456lib/Lib.java

```diff
@@ -0,0 +1,8 @@
+package issue1456lib;
+
+public class Lib {
+    public Object object;
+    public byte[] byteArray;
+    public Object object1, object2;
+    public byte byte1, byteArray2[];
```

Comment:
To test whether my other comment is true, you could add a two-dimensional array.
As a tiny comment, keeping the order of the declaration and the stubfile in sync makes them easier to read.